RecoBTag/Combined/python/pfDeepCSVJetTags_cfi.py

```diff
@@ -3,5 +3,12 @@
 pfDeepCSVJetTags = cms.EDProducer(
 	'DeepFlavourJetTagsProducer',
 	src = cms.InputTag('pfDeepCSVTagInfos'),
+  checkSVForDefaults = cms.bool(False),
+  meanPadding = cms.bool(False),
 	NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepFlavourNoSL.json')
 	)
+
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(pfDeepCSVJetTags, NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'))
+phase1Pixel.toModify(pfDeepCSVJetTags, checkSVForDefaults = cms.bool(True))
+phase1Pixel.toModify(pfDeepCSVJetTags, meanPadding = cms.bool(True))
```

Comment:
The model (line 12 and 14) has been explicitly trained for Phase I, and therefore should be used for Phase I only, there is no guarantee that the same training will perform better on Phase 0 as well.
Line 13 is a change in how the defaults are treated. We did not check the effect on the phase 0 training and we would like to keep it that way for compatibility reasons.